manifest.json

```diff
@@ -1,7 +1,10 @@
 {
+  "background": {
+    "page": "background.html"
```

Comment:
I agree to have `page` as the way we inject the script.
however, when we look at the result HTML. we can see there is bundle.js and other CSS files are injected.
which is IMO not necessary for a background script.

Reply:
@foray1010